tda/mobile_native/dotnet-maui-ios/test_checkout_dotnet_maui_ios.py

```diff
@@ -0,0 +1,27 @@
+import time
+import sentry_sdk
+import pytest
+from appium.webdriver.common.appiumby import AppiumBy
+
+@pytest.mark.skip(reason="Add to Cart stopped working on iOS")
+def test_checkout_dotnet_maui_ios(ios_dotnet_maui_driver):
+    driver = ios_dotnet_maui_driver
+
+    try:
+        driver.find_element(AppiumBy.ACCESSIBILITY_ID, "Real_World_Sample").click()
+
+        first_item_add_to_cart = driver.find_element(AppiumBy.XPATH, '(//XCUIElementTypeButton[@name="Add to Cart"])[1]')
+        first_item_add_to_cart.click()
+        # Alert disappeared for some reason on iOS
```

Comment:
Ideally we get rid of it, actually:

https://github.com/sentry-demos/dotnet-maui/issues/11

Reply:
ok, when you update the app we can change this too